view/vw_qgep_wastewater_structure.py

```diff
@@ -94,8 +94,8 @@ def vw_qgep_wastewater_structure(srid: int,
         {extra_joins};
        
         ALTER VIEW qgep_od.vw_qgep_wastewater_structure ALTER obj_id SET DEFAULT qgep_sys.generate_oid('qgep_od','wastewater_structure');
-        ALTER VIEW qgep_od.vw_qgep_wastewater_structure ALTER co_obj_id SET DEFAULT qgep_sys.generate_oid('qgep_od','structure_part');
-        ALTER VIEW qgep_od.vw_qgep_wastewater_structure ALTER wn_obj_id SET DEFAULT qgep_sys.generate_oid('qgep_od','wastewater_node');
```

Comment:
I think the second one needs to stay as "wastewater_node". As far as I understand, we want the obj id of the more specific type.

* cover is more specific than structure_part
* wastewater_node is more specific than wasterwater_networkelement

Reply:
Ok, then I need support on this.

I tried defining WN but it ended with WE prefix for vw_qgep_ws generated WN

Maybe linked to https://github.com/QGEP/QGEP/issues/536

Can you or @3nids have a look on that on Pully's behalf. It would be great to fix that quickly.

Reply:
indeed, second line is not need anymore since e4f5c8c6982467beb24489a007abdb6a3007a8d2
I will drop it and merge